scilpy/tractograms/streamline_and_mask_operations.py

```diff
@@ -128,7 +128,8 @@ def cut_outside_of_mask_streamlines(sft, binary_mask, min_len=0):
     new_streamlines = _cut_streamlines_with_masks(
         streamlines, binary_mask, binary_mask)
 
-    new_sft = StatefulTractogram.from_sft(new_streamlines, sft)
+    new_sft = StatefulTractogram.from_sft(
+        new_streamlines, sft, data_per_streamline=sft.data_per_streamline)
```

Comment:
There is no chance that the output list of streamlines contains less streamlines than the input sft? In what case copying `data_per_streamline` will not work.

Reply:
Good question... I managed it and added a logging.info to supervise this.